### PR TITLE
Go BLUI

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
-  Go BLUI
- Can someone verify this works on android/ios ?

### CODE REVIEW

**Comment by jeffvg** [comment]
looking now

**Comment by jeffvg** [comment]
looks good on iOS and not able to build for Android

image

File: config.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <widget id="io.ionic.starter" version="0.0.1" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
     <name>MyApp</name>
     <description>Reusable design patterns for ionic.</description>
-    <author email="pxblue@eaton.com" href="https://pxblue.github.io">PX Blue</author>
+    <author email="brightlayer-ui@eaton.com" href="https://brightlayer-ui.github.io">PX Blue</author>
```
Comment (jeffvg) [inline config.xml:5]: PX Blue author needs updated here

**Comment by jeffvg** [comment]
Angular Material v11, they seem to use @import syntax, and in v12 they seem to favour @use syntax. in scss but this didn't work

**Comment by emclaug2** [comment]
> Angular Material v11, they seem to use @import syntax, and in v12 they seem to favour @use syntax. in scss but this didn't work

This demo is still using ng11 & I'm hoping it'll stay that way forever. 👍

File: config.xml (hunk 1 above)
Comment (emclaug2) [reply config.xml]: Good catch!

**Comment by jeffvg** [comment]
looking at android now

**Comment by jeffvg** [comment]
still having issues building Android on my system.

image

image

**Comment by andrupu** [comment]
Looks like that is a Yarn issue.  Try running 'yarn' before running 'ionic cordova build'.

**Comment by emclaug2** [comment]
@jeffvg This is a weird error; if it persists after running yarn let me know.  Maybe there's a missing assets specified in the config _somewhere_.

**Comment by jeffvg** [comment]
> @jeffvg This is a weird error; if it persists after running yarn let me know. Maybe there's a missing asset in a config _somewhere_.

yarn install was run prior to this. If not cordova cmds do not work. I'll try on fresh clone

**Comment by jeffvg** [comment]
maybe someone else can verify android? this could be my mac env. too
I'm running this...

yarn

ionic cordova resources android
ionic cordova platform add android
ionic cordova build android

**Comment by emclaug2** [comment]
I got it to work on android on my windows PC, but I noticed that the app bar was not stickied to the top of the app.  

I'll open an issue for this.

**Comment by emclaug2** [comment]
> I'll open an issue for this.

Nevermind, there is already an issue for this, #27 

@jeffvg   This should be good to go since you tested on ios

**Comment by emclaug2** [comment]
>  ionic cordova resources android ionic cordova platform add android ionic cordova build android

These commands look good to me, I also deleted my node modules & ran yarn before doing the above commands.

**Comment by jeffvg** [comment]
ok so I'm running now.
1. fresh clone
2. ionic cordova resources android
3. ionic cordova platform add android
4. yarn
5. ionic cordova build android
6. ionic cordova run android
7. it's running

image

image

**Review by jeffvg — APPROVED** [review]

**Pull request merged**